components/layout/navbar.module.css

```diff
@@ -1,19 +1,40 @@
 .container {
   display: flex;
-  justify-content: flex-end;
-  margin: 5px 10px;
+  padding: 1rem;
+  justify-content: space-between;
 }
 
 .avatar {
   box-shadow: rgba(0, 0, 0, 0.19) 0px 10px 20px, rgba(0, 0, 0, 0.23) 0px 6px 6px;
   cursor: pointer;
+  background: #1c6759 !important;
```

Comment:
Bad practice, use the chakra ui property.

Reply:
Listo